docs-i18n-tracker/lib/translation-status/template.html

```diff
@@ -0,0 +1,267 @@
+		}
+
+		h2 {
+			font-size: 1.875rem;
+		}
+
+		@media (min-width: 37.75em) {
+			h1 {
+				font-size: 2.5rem;
+			}
+		}
+
+		main {
+			max-width: 80ch;
+			margin-inline: auto;
+		}
+
+		.limit-to-viewport {
+			max-width: calc(100vw - 2rem);
+		}
+
+		p+p {
+			margin-top: 1.25rem;
+		}
+
+		a {
+			color: var(--theme-accent);
+			text-decoration: none;
+		}
+
+		h2 a {
+			color: inherit;
+		}
+
+		p a {
+			color: var(--theme-accent);
+			text-decoration: underline;
+		}
+
+		a:hover {
+			text-decoration: underline;
+			color: var(--theme-text-bright);
+		}
+
+		ul {
+			font-size: 0.875rem;
+		}
+
+		details {
+			margin-bottom: 1.25rem;
+		}
+
+		details summary {
+			cursor: pointer;
+			user-select: none;
+		}
+
+		details summary:hover strong {
+			color: var(--theme-accent-secondary);
+		}
+
+		details h5 {
+			margin-top: 1rem !important;
+		}
+
+		details> :last-child {
+			margin-bottom: 1rem;
+		}
+
+		.create-button {
+			padding: 0.1em 0.5em;
+			background-color: hsl(213deg 89% 64% / 20%);
+			display: inline-block;
+			border-radius: 0.5em;
+			font-weight: bold;
+			font-size: 0.75rem;
+		}
+
+		.status-by-page {
+			margin-bottom: 1rem;
+			border-collapse: collapse;
+			margin-inline: -1rem;
+			border: var(--theme-table-border);
+			font-size: 0.8125rem;
+			width: 85%;
+		}
+
+		.table-container {
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+			align-items: center;
+			margin-top: 30px;
+			width: 100%;
+			gap: 5px;
+		}
+
+		.status-by-page th {
+			position: sticky;
+			top: -1px;
+			background: var(--theme-table-header);
+			white-space: nowrap;
+			padding-inline: 0.3rem;
+		}
+
+		.status-by-page th,
+		.status-by-page td {
+			padding-block: 0.2rem;
+		}
+
+		.status-by-page tbody tr:not(.spacer):hover td {
+			background: var(--theme-table-hover);
+		}
+
+		.status-by-page .spacer td {
+			height: 0.5rem;
+		}
+
+		.status-by-page th:first-of-type,
+		.status-by-page td:first-of-type {
+			text-align: left;
+			padding-inline-start: 1rem;
+		}
+
+		.status-by-page th:last-of-type,
+		.status-by-page td:last-of-type {
+			text-align: center;
+			padding-inline-end: 1rem;
+		}
+
+		.status-by-page td:not(:first-of-type) {
+			min-width: 2rem;
+			text-align: center;
+			cursor: default;
+		}
+
+		.status-by-page td:not(:first-of-type) a {
+			text-decoration: none;
+		}
+
+		.progress-summary {
+			font-size: 0.8125rem;
+		}
+
+		.progress-bar {
+			font-size: 0.6875rem;
+		}
+
+		.main-container {
+			position: relative;
+		}
+	</style>
+</head>
+
+<body>
+	<main class="main-container">
+		<div class="limit-to-viewport">
+			<h1>Starlight Docs Translation Status</h1>
+
+			<p>
+				If you're interested in helping us translate
+				<a href="https://starlight.astro.build/">starlight.astro.build</a> into one of the languages listed
+				below, you've come to the right place! This auto-updating page always lists all the
+				content that could use your help right now.
+			</p>
```

Comment:
```suggestion
			<p>
				If you're interested in helping us translate
				<a href="https://starlight.astro.build/">starlight.astro.build</a> into one of the languages listed
				below, you've come to the right place! This auto-updating page always lists all the
				content that could use your help right now.
			</p>
			<p>
				Before starting a new translation, be sure to check the [existing Starlight PRs](https://github.com/withastro/starlight/pulls) to see if this page has already been translated, and consider reviewing any open PRs in your language!
			</p>
```
Thoughts? Maybe we should add something like this to Astro docs translation status page, too!